pkg/resourceinterpreter/interpreter.go

```diff
@@ -99,6 +99,7 @@ func (i *customResourceInterpreterImpl) Start(ctx context.Context) (err error) {
 	i.defaultInterpreter = native.NewDefaultInterpreter()
 
 	i.informer.Start()
+	i.informer.WaitForCacheSync()
```

Comment:
This line does not seem to be directly related to the `ResyncPeriod`

Reply:
So, what's your option?

I guess it's necessary, right?